rc/filetype/kotlin.kak

```diff
@@ -0,0 +1,277 @@
+
+  # ------------------------------------------------------------------------------------------------- #
+
+  ktor_types='ApplicationCall ApplicationSendPipeline ByteReadChannel Headers HeadersBuilder HttpClientCall HttpClientEngine HttpMessageBuilder HttpMethod HttpProtocolVersion HttpRequestBuilder HttpResponse HttpStatusCode ResponseCookies ResponseHeaders URLBuilder'
+
+  # ------------------------------------------------------------------------------------------------- #
+
+  kdocs='author constructor exception param property receiver return sample see since suppress throws'
+
+  # ------------------------------------------------------------------------------------------------- #
+
+  kotlin_errors_ALL='Error AssertionError NotImplementedError OutOfMemoryError'
+
+  kotlin_exceptions_ALL='CharacterCodingException IllegalCallableAccessException IllegalPropertyDelegateAccessException NoSuchPropertyException RuntimeException Throwable'
+
+  # ------------------------------------------------------------------------------------------------- #
+
+  java_errors_ALL='AbstractMethodError AnnotationFormatError AssertionError AWTError BootstrapMethodError ClassCircularityError ClassFormatError CoderMalfunctionError ExceptionInInitializerError FactoryConfigurationError GenericSignatureFormatError IllegalAccessError IncompatibleClassChangeError InstantiationError InternalError IOError JMXServerErrorException​ LinkageError NoClassDefFoundError NoSuchFieldError NoSuchMethodError OutOfMemoryError RuntimeErrorException RuntimeErrorException​ SchemaFactoryConfigurationError ServerError​ ServiceConfigurationError StackOverflowError ThreadDeath TransformerFactoryConfigurationError UnknownError UnsatisfiedLinkError UnsupportedClassVersionError VerifyError VirtualMachineError ZipError'
+
+  # ------------------------------------------------------------------------------------------------- #
+
+  java_exceptions_A='AbsentInformationException AcceptPendingException AccessControlException AccessDeniedException AccessException AccountException AccountExpiredException AccountLockedException AccountNotFoundException ActivateFailedException ActivationException AEADBadTagException AgentInitializationException AgentLoadException AlreadyBoundException AlreadyBoundException AlreadyConnectedException AnnotationTypeMismatchException ArithmeticException ArrayIndexOutOfBoundsException ArrayStoreException AsynchronousCloseException AtomicMoveNotSupportedException AttachNotSupportedException AttachOperationFailedException AttributeInUseException AttributeModificationException AttributeNotFoundException AuthenticationException AuthenticationException AuthenticationNotSupportedException AWTException'
```

Comment:
This should be a single list and there should be a single highlighter for those exception names.